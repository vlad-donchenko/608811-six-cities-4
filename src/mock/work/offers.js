const offers = [
  {
    title: `Beautiful & luxurious apartment at great location`,
    picture: `img/apartment-01.jpg`,
    price: 120,
    type: `Apartment`,
    rating: 4,
    isPremium: true,
    isBookmark: true
  },
  {
    title: `Wood and stone place`,
    picture: `img/apartment-01.jpg`,
    price: 220,
    type: `Apartment`,
    rating: 3,
    isPremium: false,
    isBookmark: false
  },
  {
    title: `Canal View Prinsengracht`,
    picture: `img/apartment-01.jpg`,
    price: 150,
    type: `Apartment`,
    rating: 2,
    isPremium: false,
    isBookmark: false
  },
  {
    title: `Nice, cozy, warm big bed apartment`,
    picture: `img/apartment-01.jpg`,
    price: 850,
    type: `Apartment`,
    rating: 5,
    isPremium: false,
    isBookmark: true
  },
  {
    title: `Wood and stone place`,
    picture: `img/apartment-01.jpg`,
    price: 450,
    type: `Apartment`,
    rating: 4.5,
    isPremium: true,
    isBookmark: false
  }
];

export {offers};
