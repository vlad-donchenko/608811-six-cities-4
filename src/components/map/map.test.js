import React from 'react';
import renderer from 'react-test-renderer';
import Map from "./map";
import {offers} from "../../mock/test/offers";

it(`MapCities is rendered correctly`, () => {
  Map.prototype.componentDidMount = jest.fn();
  const tree = renderer.create(
    <Map
      offers={offers}
    />,
    {
      createNodeMock: () => {
        return document.createElement(`div`);
      }
    }).toJSON();

  expect(tree).toMatchSnapshot();
});
