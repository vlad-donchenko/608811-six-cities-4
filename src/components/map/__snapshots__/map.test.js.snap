// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`MapCities is rendered correctly 1`] = `
<div
  id="map"
  style={
    Object {
      "height": "100%",
    }
  }
/>
`;
